application/src/main/java/org/togetherjava/tjbot/commands/UserInteractor.java

```diff
@@ -66,6 +67,24 @@ public interface UserInteractor extends Feature {
      */
     void onSelectionMenu(SelectMenuInteractionEvent event, List<String> args);
 
+    /**
+     * Triggered by the core system when a modal corresponding to this implementation (based on
+     * {@link #getName()}) has been clicked.
+     * <p>
+     * This method may be called multithreaded. In particular, there are no guarantees that it will
+     * be executed on the same thread repeatedly or on the same thread that other event methods have
+     * been called on.
+     * <p>
+     * Details are available in the given event and the event also enables implementations to
+     * respond to it.
+     *
+     * @param event the event that triggered this
+     * @param args the arguments transported with the selection menu, see
+     *        {@link SlashCommand#onSlashCommand(SlashCommandInteractionEvent)} for details on how
+     *        these are created
+     */
+    void onModal(ModalInteractionEvent event, List<String> args);
```

Comment:
`onModal`-what? its missing some verb. like `onModalReceived`. or maybe even better, to be consistent with JDA `onModalInteraction`.

Reply:
fyi, the other methods are onSelectionMenu and onButtonClick

So if we're gonna name this `onModalInteraction`, the others should be too

Reply:
`onButtonClick` is good. but `onSelectionMenu` is indeed bad as well.

Reply:
modified all 3 methods to be consistent and use Interaction